src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -167,6 +252,31 @@ public static async Task<WwwAuthenticateParameters> CreateFromResourceResponseAs
             return wwwAuthParam;
         }
 
+        /// <summary>
+        /// Create the authenticate parameters by attempting to call the resource unauthenticated, and analyzing the response.
+        /// </summary>
+        /// <param name="httpClient">Instance of <see cref="HttpClient"/> to make the request with.</param>
+        /// <param name="resourceUri">URI of the resource.</param>
+        /// <param name="cancellationToken">The cancellation token to cancel operation.</param>
+        /// <param name="scheme">Authentication scheme. Default is Bearer.</param>
+        /// <returns>WWW-Authenticate Parameters extracted from response to the unauthenticated call.</returns>
+        public static async Task<IEnumerable<WwwAuthenticateParameters>> CreateAllFromResourceResponseAsync(HttpClient httpClient, string resourceUri, CancellationToken cancellationToken = default, string scheme = "Bearer")
```

Comment:
Use `Constants.BearerAuthHeaderPrefix`

Reply:
Actually, this scheme param is here in error, need to be remove